### PR TITLE
Update Process for Instating Trusted Committers

### PR DESCRIPTION
**Pull request by maxcapraro**
This pull request proposes ...
- A lean process to instate trusted committers for the patterns working group
- To instate @lenucksi as an additional trusted committer

In a call earlier today, @spier, @lenucksi, @mainec, and I felt that this will be benficial for multiple reasons:
- Having a dedicated mandate from the current trusted committers, would allow us to revive the working group and bring in our ideas, without the fear of trampling anyones feet
- Having a (or a set of) trusted committers who currently want to and can commit time, will make sure that contributions (particularly those of new contributors) will be seen to quickly - motivating them to stay
- Having an explicit process for instating new trusted committers, will make the instatement (and the responsibilities it brings) explicit
- Having an explicit process enables us to motivate contributors; and reward their contributions

### CODE REVIEW

**Comment by maxcapraro** [comment]
1. @nyeates, @gruetter, @NewMexicoKid, @cewilliams you were listed explicitly as repo Admins. This is why I added you to the TC list. Please let me know if you have thoughts about this.
2. What do we do with the many folks who have earned write access in the past? Do we simple want to add them to the TC list? Revoke write access? Let them their write acces but not add them to the list?

**Comment by spier** [comment]
Looks good to me.

I think you would also have to change the link to `TRUSTED-COMMITTERS.md` from within `README.md`?

**Comment by maxcapraro** [comment]
> I think you would also have to change the link to TRUSTED-COMMITTERS.md from within README.md?

Thanks .... Indeed! I'll do that tomorrow :)

**Comment by maxcapraro** [comment]
Tomorrow became today. Fixed it (and created #142 to in hopes to avoid this in the future).

**Comment by MaineC** [comment]
> What do we do with the many folks who have earned write access in the past? Do we simple want 
> to  add them to the TC list? Revoke write access? Let them their write acces but not add them to 
> the list?

I would ask them. That gives us an excuse to get in touch with them and talk about breathing more life into the patterns community - and it gives them the option to make a conscious decision.

If you need a template text to draft such an invitation:
http://community.apache.org/newcommitter.html#committer-invite-template (note to self: [This](https://tech.europace.de/voting-in-new-trusted-committers/) needs to be turned into another InnerSource pattern.)

File: README.md
Comment (rrrutledge) [inline README.md]: Broken link.  Where is it intending to point?

File: TRUSTED-COMMITTERS.md
Comment (rrrutledge) [inline TRUSTED-COMMITTERS.md]:
```suggestion
   * New TC is praised in the [#innersource-patterns](https://app.slack.com/client/T04PXKRM0/C2EFRTS6A) channel.
```

I don't think that the entire commons needs to be notified - just the people that are interested in the InnerSource Patterns working group.

**Review by rrrutledge — CHANGES_REQUESTED** [review]
Thanks for your all work, here.

File: README.md
Comment (maxcapraro) [reply README.md]:
```suggestion
* [Trusted Committer](TRUSTED-COMMITTERS.md) - *Many inner-source projects will find themselves in a situation where they consistently receive feedback, features, and bug-fixes from contributors. In these situations project maintainers seek ways to recognize and reward the work of the contributor above and beyond single contributions.*
```

Should point to the newly introduced file ... :-) Fixed the typo. Thanks @rrrutledge :)

File: TRUSTED-COMMITTERS.md
Comment (maxcapraro) [reply TRUSTED-COMMITTERS.md]: I agree with the proposed change (and already committed it to the PR branch). 

For documentation purpose: Our idea was to give praise to a wide audience, also to recognize (and motivate) new committers. I agree though, for the beginning, it will be likely be sufficient to announce it in the patterns channel.

**Comment by maxcapraro** [comment]
> I would ask them. [...]

Excellent! I agree :) Let's do that. 

This does not need to stop this PR though, or does it?

File: README.md
Comment (rrrutledge) [inline README.md]: Actually I don't think that you should change this link.  The original link describes the pattern of _having a trusted committer in the first place_.  You are changing this to link to point to _the trusted committers for this project_.  In this part of the README I think that you want the original link.  It is the part of the README that is just linking to all kinds of patterns in general, rather than information about how the repo/projects are managed.

Comment (maxcapraro) [reply README.md]: Yikes. You are of course right. The wrong link was exchanged.

Comment (spier) [reply README.md]: Great catch @rrrutledge !

**Review by rrrutledge — APPROVED** [review]

File: TRUSTED-COMMITTERS.md
Comment (lenucksi) [inline TRUSTED-COMMITTERS.md:5]: we could also think about referring to a `.github/codeowners` file with the listed persons handles in it. 
That would this would give [a bit of automation](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners).

**Review by lenucksi — COMMENTED** [review]
This looks like a great PR to me.

File: TRUSTED-COMMITTERS.md
Comment (spier) [reply TRUSTED-COMMITTERS.md:5]: Just do it. :)
CODEOWNERS are awesome, and it is a cheap experiment.

Comment (lenucksi) [reply TRUSTED-COMMITTERS.md:5]: I'll try that once this gets merged.

**Comment by maxcapraro** [comment]
I merged this PR because:
- @NewMexicoKid (founder of the working group) expressed he is happy that folks want to continue working with the patterns contents: https://innersourcecommons.slack.com/archives/C2EFRTS6A/p1585680572052500
- With @rrrutledge one person who was not part of the "re-animation meeting" agreed to the merge
- Nobody disagreed to the PR in 14 days

File: TRUSTED-COMMITTERS.md
Comment (spier) [inline TRUSTED-COMMITTERS.md:37]: @maxcapraro do you want to keep this headline? That section doesn't have any content, so when looking at the rendered HTML it looks like there is something missing. I can send a PR to remove the section if you like.

Comment (maxcapraro) [reply TRUSTED-COMMITTERS.md:37]: Nope. Thank you :) I deleted it.

**Pull request merged**
